worker/src/main/java/gov/cms/ab2d/worker/adapter/bluebutton/BfdClientAdapter.java

```diff
@@ -0,0 +1,11 @@
+package gov.cms.ab2d.worker.adapter.bluebutton;
+
+import java.nio.file.Path;
+import java.util.concurrent.Future;
+import java.util.concurrent.locks.ReentrantLock;
+
+public interface BfdClientAdapter {
```

Comment:
We should probably rename this interface and the corresponding implementation class to more accurately reflect its purpose, since it does more than just "adapting" the BFD client. `PatientClaimsProcessor` or something along these lines would do it.

Reply:
done